djangocms_blog/models.py

```diff
@@ -232,7 +232,7 @@ def get_title(self):
         return title.strip()
 
     def get_keywords(self):
-        return self.safe_translation_getter('meta_keywords').strip().split(',')
+        return self.safe_translation_getter('meta_keywords', default='').strip().split(',')
```

Comment:
ps: btw, how can the error be reproduced? Because here: https://github.com/nephila/djangocms-blog/blob/develop/djangocms_blog/models.py#L133  "meta_keywords" have the '' default.
Just to have an idea of the flow error. Anyway, it's a sane check.